### ISSUE TITLE
[issue]: (No bootfile found for UEFI! Maybe the image does not support X64 UEFI) Even though though the ISO is on their support list

### ISSUE THREAD

**Issue by aeclgithub** [ventoy/Ventoy#1270]
### Official FAQ

- [X] I have checked the official FAQ.

### Ventoy Version

1.0.61

### What about latest release

Yes. I have tried the latest release, but the bug still exist.

### BIOS Mode

Both

### Partition Style

MBR

### Disk Capacity

8GB

### Disk Manufacturer

Asaki

### Image file checksum (if applicable)

Yes.

### Image file download link (if applicable)

https://archcraft.io/download.html / http://mirror.rackspace.com/ archlinux/iso/2021.11.01/archlinux-2021.11.01-x86_64.iso

### What happened?

Disk Images: Archcraft and Arch Linux
(No bootfile found for UEFI! Maybe the image does not support X64 UEFI) 

Used to work flawlessly the first 2 days.

**Comment by ventoy** [comment]
It used to boot OK with the same ISO file in the same USB stick pluged in the same USB port of the same computer? right?
1. press `m` to checksum the ISO file to make sure that it is not corrupted
2. Test in different computers

**Comment by aeclgithub** [comment]
> It used to boot OK with the same ISO file in the same USB stick pluged in the same USB port of the same computer? right?
> 
> 1. press `m` to checksum the ISO file to make sure that it is not corrupted
> 2. Test in different computers

So sorry but could you please explain about the `m` part?

**Comment by aeclgithub** [comment]
And I have checked the sum if that's what  its applying

**Comment by ventoy** [comment]
> > It used to boot OK with the same ISO file in the same USB stick pluged in the same USB port of the same computer? right?
> > 
> > 1. press `m` to checksum the ISO file to make sure that it is not corrupted
> > 2. Test in different computers
> 
> So sorry but could you please explain about the `m` part?

https://www.ventoy.net/en/doc_checksum.html

**Comment by aeclgithub** [comment]
> > > It used to boot OK with the same ISO file in the same USB stick pluged in the same USB port of the same computer? right?
> > > 
> > > 1. press `m` to checksum the ISO file to make sure that it is not corrupted
> > > 2. Test in different computers
> > 
> > 
> > So sorry but could you please explain about the `m` part?
> 
> https://www.ventoy.net/en/doc_checksum.html

Alrighty it came with an output but same error

**Comment by aeclgithub** [comment]
Alright then.

**Comment by stepanmas** [comment]
Same issue.

**Comment by steve6375** [comment]
archlinux-2021.11.01-x86_64.iso is latest

UEFI64
![image](https://user-images.githubusercontent.com/1469350/155102902-d90e3565-46d1-449f-a72c-0db81ea63667.png)

**Comment by AchmadFathoni** [comment]
Still doesn't work, Archlinux 2022.02.01 MBR NTFS UEFI

**Comment by arafatx** [comment]
Well, all of these ISO files (taken from their official sites) and I tried to boot each of them:

![image](https://user-images.githubusercontent.com/4420157/160265347-0ef882f7-bca3-4b5d-bd23-cf9d84103172.png)


and each of them came with this message:

```
No bootfile found for UEFI! Maybe the image does not support X64 UEFI

```

Information:

- [ ] My thumb drive is 32GB

- [ ] Tried using GPT / MBR partition style.

- [ ] Tried the option secure boot support and also tried without it.

- [ ] The thumb drive is plugged into USB port 3.0 (I notice that it will not detect my USB port if I put it on USB port 2.0)


Saw on youtube that ventoy is easy to use with just copying and pasting the ISO. I don't think so. Most of the ISO files I tried above were tested as claimed.

**Comment by AchmadFathoni** [comment]
> Still doesn't work, Archlinux 2022.02.01 MBR NTFS UEFI
I got a working case
Ventoy : 1.0.72
ISO : archlinux-2022.03.01-x86_64.iso
Flashdisk : ffff:5678 USB Disk 2.0
Partition : MBR or GPT with exFAT
Laptop : 20TLS13100 ThinkPad X1 Extreme Gen 3

**Comment by arafatx** [comment]
My bad from the previous post. Actually, I created the bootable disk manually using a command prompt. I was trying to play around with it and I did not realize that it was a custom bootable disk. Now everything seems ok.

**Comment by liudonghua123** [comment]
I have the same issue when i tried the win7 32bit iso, win7 64bit iso is no problem. I use the latest ventoy.

**Comment by ventoy** [comment]
> I have the same issue when i tried the win7 32bit iso, win7 64bit iso is no problem. I use the latest ventoy.

32位的Windows系统本来就不能在64位UEFI模式下安装，你这个不算问题。

**Comment by liudonghua123** [comment]
> > I have the same issue when i tried the win7 32bit iso, win7 64bit iso is no problem. I use the latest ventoy.
> 
> 32位的Windows系统本来就不能在64位UEFI模式下安装，你这个不算问题。
> 

默认uefi都是64位的？那我如果想用ventoy引导安装win7 32位的系统，我应该在bios里面禁用uefi引导，然后重新从U盘引导就可以了？但我记得我bios里面一直用的都是legacy bios模式的呀😅

**Comment by yufei8840** [comment]
请问一下，为啥经过测试的Win7x64今天在笔记本安装的时候，也是同样的错误。

**Comment by ZhiboWong** [comment]
我今天也遇到了此问题。U盘重做ventoy，pe换成wepe就好了。也不知道啥原因导致的故障。

**Comment by Stalinist1** [comment]
I have the same issue.

**Comment by steve6375** [comment]
Is your system BIOS a UEFI32 firmware or UEFI64 firmware? This does not depend on the CPU (for instance some systems have a 32-bit firmware but a 64-bit CPU.
So you need to specify the EXACT system you are using and the EXACT ISO and the EXACT MD5 checksum (obtained by pressing m key in Ventoy).

**Comment by Stalinist1** [comment]
> So you need to specify the EXACT system you are using and the EXACT ISO and the EXACT MD5 checksum (obtained by pressing m key in Ventoy).

I download sha256 file with checksum for every ISO I download. My files are OK if you think they downloaded corrupted.

**Comment by steve6375** [comment]
No, the file you downloaded is probably OK but the file on the USB drive could be corrupt so you need to check the file that is on your USB drive.
Also, you still haven't answered my three Q's?

**Comment by Stalinist1** [comment]
> Also, you still haven't answered my three Q's?

> Is your system BIOS a UEFI32 firmware or UEFI64 firmware? 

I don't know. I bought my mini-PC this summer.

**Comment by steve6375** [comment]
Unfortunately, I was not there when you bought it, so I don't know the make and model of your new mini-PC and my crystal ball is broken ;-)
If I knew the EXACT make and model, I could look it up on this new thing called _Google_ and find out for you,

**Comment by Stalinist1** [comment]
> Unfortunately, I was not there when you bought it

Very Funny (No)
You can examine it. https://aliexpress.ru/item/1005005317467233.html

**Comment by steve6375** [comment]
Ok, and give one specific iso that wont boot?

**Comment by Stalinist1** [comment]
> Ok, and give one specific iso that wont boot?

https://www.slax.org/

**Comment by zdian** [comment]
> No bootfile found for UEFI!
> Maybe the image does not support X64 UEFI

 I got the same error when I use the latest “zh-cn_windows_11_business_editions_version_23h2_updated_july_2024_x64_dvd_8c3fbaa8.iso”
  But “zh-cn_windows_11_business_editions_version_23h2_updated_june_2024_x64_dvd_d5833049.iso” or other iso is work fine.
